exercises/F-predicates/exercise.js

```diff
@@ -5,11 +5,23 @@
   The variables should have values that match the expected results.
 */
 
+function isSmall(value) {}
+
+function isNegative(x) {}
+
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  if (number >= 0 && number <= 10) {
```

Comment:
Hi Hamra. Good job it is very easy to understand the code.I did it in another way like this:
 return (number > 0 && number < 10);